src/Logging.AzureAppServices/test/ConfigureOptionsTests.cs

```diff
@@ -21,7 +21,7 @@ public void InitializesIsEnabled(bool? enabled)
         {
             var configuration = new ConfigurationBuilder().AddInMemoryCollection(new[]
             {
-                new KeyValuePair<string, string>("IsEnabledKey", Convert.ToString(enabled))
+                new KeyValuePair<string, string>("IsEnabledKey", Convert.ToString(enabled, null))
```

Comment:
TIL that null is an option here. I've always passed `Culture.InvariantCulture`.

Reply:
Looks like dotnet/runtime goes a bit deeper, to `NumberFormatInfo` values e.g.
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime.Extensions/tests/System/Convert.ToString.cs#L485

Suggest `NumberFormatInfo.InvariantInfo` here

Reply:
It goes through https://github.com/dotnet/runtime/blob/e5dd7150e6ced783159a8ae3adb77b899b1204db/src/libraries/System.Private.CoreLib/src/System/Convert.cs#L1942, which means it doesn't matter.  Going to leave it null